docs/dev.md

```diff
@@ -20,14 +20,45 @@ Dev mode provides three key features. Code changes are detected, recompiled, and
 
 The following are dev mode supported code changes. Changes to your server such as changes to the port, server name, hostname, etc. will require restarting dev mode to be detected.  Changes other than those listed below may also require restarting dev mode to be detected.
 
-* Java source file changes and Java test file changes are detected, recompiled, and picked up by your running server.  
+* Java source file changes and Java test file changes are detected, recompiled, and picked up by your running server.
 * Added dependencies to your `pom.xml` are detected and added to your classpath.  Dependencies that are Liberty features will be installed via the `install-feature` goal.  Any other changes to your `pom.xml` will require restarting dev mode to be detected.
 * Resource file changes are detected and copied into your `target` directory. 
 * Configuration directory and configuration file changes are detected and copied into your `target` directory, which are hot deployed to the server.  Added features to your `server.xml` will be installed and picked up by your running server.  Adding a configuration directory or configuration file that did not previously exist while dev mode is running will require restarting dev mode to be detected.
 
+###### Multiple Modules
+
+Dev mode can be run on a single Maven module or on a multi module Maven project (a project consisting of multiple modules specified in the `<modules>` section of its `pom.xml`).  When run on a single Maven module, only changes within that module are detected and hot deployed.  When run on a multi module Maven project, changes in all modules are detected and hot deployed according to the Maven Reactor build order.  Note that any modules that other modules rely on as a compile dependency must have a non-empty Java source folder with Java file(s) before starting dev mode, otherwise the other modules may fail to compile.
+
+If you want to start dev mode with the short-form `liberty` name for the Liberty Maven plugin, do one of the following:
+1. In `~.m2/settings.xml`, add:
+```
+<pluginGroups>
+  <pluginGroup>io.openliberty.tools</pluginGroup>
+</pluginGroups> 
+```
+2. or define the Liberty Maven plugin in the parent pom.xml of every module,  
+3. or define the Liberty Maven plugin in pom.xml of every module.
+
+Then from the directory containing the multi module `pom.xml`, run:
+```
+$ mvn liberty:dev
+```
+
+Otherwise, from the directory containing the multi module `pom.xml`, run:
+```
+$ mvn io.openliberty.tools:liberty-maven-plugin:3.4:dev
+```
+
+Liberty server configuration files (such as server.xml) will be used from the module that does not have any other modules depending on it.  If there is more than one module without other modules depending on it, specify which module with Liberty configuration that you want to use by including the parameters `-pl <module-with-liberty-config> -am`.  
```

Comment:
server.xml should be in quotes to keep consistency in the document (`server.xml`)